app/views/katello/api/v2/activation_keys/base.json.rabl

```diff
@@ -7,6 +7,30 @@ node :multi_content_view_environment do |ak|
   ak.multi_content_view_environment?
 end
 
+child :content_view_environments => :content_view_environments do
+  node :content_view do |cve|
+    {
+      id: cve.content_view&.id,
+      name: cve.content_view&.name,
+      composite: cve.content_view&.composite,
+      content_view_version: cve.content_view_version&.version,
+      content_view_version_id: cve.content_view_version&.id,
+      content_view_version_latest: cve.content_view_version&.latest?,
+      content_view_default: cve.content_view&.default?
+    }
+  end
+  node :lifecycle_environment do |cve|
+    {
+      id: cve.lifecycle_environment&.id,
+      name: cve.lifecycle_environment&.name,
+      lifecycle_environment_library: cve.lifecycle_environment&.library?
+    }
+  end
+  node :candlepin_name do |cve|
```

Comment:
see https://github.com/Katello/katello/pull/11161#discussion_r1786750641

Reply:
Ok I'll change it to katello_name

Reply:
The options are either `label` or `candlepin_name`. I think we were leaning towards `label` but wanted to get a consensus first

Reply:
Label sounds better than candlepin name and does not expose the internal product like Samir was saying.

Reply:
Maybe environment label?

Reply:
+1 for label then

Reply:
Okay, let's go with that :+1:  You can change the name of the node here, and I will change the internal method names in my followup.

Reply:
Updated to label